eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -73,9 +74,13 @@ foreach ($rg in $toDelete)
 {
   if ($Force -or $PSCmdlet.ShouldProcess("$($rg.ResourceGroupName) (UTC: $($rg.Tags.DeleteAfter))", "Delete Group")) {
     # Add purgeable resources that will be deleted with the resource group to the collection.
-    $purgeableResources += Get-PurgeableGroupResources $rg.Name
+    $purgeableResourcesFromRG = Get-PurgeableGroupResources $rg.ResourceGroupName
 
-    Write-Verbose "Deleting group: $($rg.Name)"
+    if ($purgeableResourcesFromRG -and $purgeableResourcesFromRG.Count -gt 0) {
```

Comment:
Nit: The count is unnecessary. PowerShell is truthy, which means an undefined collection variable == null collection == an empty collection.